### PR TITLE
Support for JaCoCo XML input #79

### PR DESCRIPTION
**Pull request by markbates**

### CODE REVIEW

File: cmd/format-coverage.go [hunk 1]
```diff
@@ -32,7 +33,7 @@ type CoverageFormatter struct {
 var formatOptions = CoverageFormatter{}
 
 // a prioritized list of the formatters to use
-var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura"}
+var formatterList = []string{"simplecov", "lcov", "coverage.py", "clover", "gocov", "gcov", "cobertura", "jacoco"}
```
Comment (GordonDiggs) [inline cmd/format-coverage.go]: Don't need to do it in this PR, but could we alphabetize this list and the Map below? Would that hurt?

Comment (markbates) [inline cmd/format-coverage.go]: We could, but that is a prioritized look up list. Sorting it alphabetically would change how we look up for matters. That was a requirement early on. If it's no longer a requirement we can delete the list altogether. :)

File: formatters/simplecov/simplecov.go [hunk 2]
```diff
@@ -47,8 +47,8 @@ func (r Formatter) Format() (formatters.Report, error) {
 		return rep, errors.WithStack(err)
 	}
 
+	gitHead, _ := env.GetHead()
```
Comment (GordonDiggs) [inline formatters/simplecov/simplecov.go]: What's this change? Does it belong in this PR?

File: cmd/format-coverage.go (hunk 1 above)
Comment (GordonDiggs) [inline cmd/format-coverage.go]: Got it - wasn't sure if it was prioritized. It's fine for now!

**Pull request merged**